### PR TITLE
[k8s-configuration] Release v1.3.0 with deployed-object support

### PR DESCRIPTION
**Pull request by jonathan-innis**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
k8s-configuration

File: src/k8s-configuration/azext_k8s_configuration/_help.py [hunk 1]
```diff
@@ -174,3 +174,29 @@
           --cluster-name mycluster --cluster-type connectedClusters --name myconfig \\
           --kustomization-name my-kustomization
 """
+
+helps['k8s-configuration flux deployed-object'] = """
+    type: group
+    short-summary: Commands to see deployed objects associated with Flux v2 Kubernetes configurations.
```
Comment (wangzelin007) [inline src/k8s-configuration/azext_k8s_configuration/_help.py:180]: May I ask you to use the consistent description ？
`Flux v2 Kubernetes configurations` in line 180.
`Kubernetes Flux v2 Configuration` in line 185, 187, 197.
`Flux v2 Configuration` in line 195.

Comment (jonathan-innis) [reply src/k8s-configuration/azext_k8s_configuration/_help.py:216]: Fixed

**Comment by jonathan-innis** [comment]
@zhoxing-ms @wangzelin007 Can you please review/merge this one?

File: src/k8s-configuration/azext_k8s_configuration/_params.py [hunk 2]
```diff
@@ -118,6 +118,15 @@ def load_arguments(self, _):
                    options_list=['--yes', '-y'],
                    help='Do not prompt for confirmation')
 
+    with self.argument_context('k8s-configuration flux deployed-object show') as c:
+        c.argument('object_name',
+                   help='Name of the object deployed by the configuration on the cluster.')
+        c.argument('object_namespace',
+                   help='Namespace of the object deployed by the configuration on the cluster.')
+        c.argument('object_kind',
+                   arg_type=get_enum_type(['GitRepository', 'Bucket', 'HelmRepository', 'HelmChart', 'HelmRelease', 'Kustomization']),
```
Comment (zhoxing-ms) [inline src/k8s-configuration/azext_k8s_configuration/_params.py:127]: May I ask could we get those enumeration values from the Python SDK?

Comment (jonathan-innis) [reply src/k8s-configuration/azext_k8s_configuration/_params.py:127]: We do not currently have these types defined in the swagger but will define them at the next iteration

**Comment by jonathan-innis** [comment]
@zhoxing-ms Can you please re-review?

**Review by wangzelin007 — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**